sage-ios/SAGE/SAGE/AdminOperations.swift

```diff
@@ -54,6 +54,12 @@ class AdminOperations {
         
     }
     
+    static func createAnnouncement(announcement: Announcement, completion: (Announcement) -> Void, failure: (String) -> Void) {
+        let manager = BaseOperation.manager()
+        let announcementDict = announcement.toDictionary()
+        // TODO: Create Announcement
```

Comment:
should i add to this pr or leave for later? i didnt even try making network requests lul i was thinking i'd make like 6 of them as the next pr